docs/networks/mainnet/lukso-mainnet.md

```diff
@@ -0,0 +1,228 @@
+
+## LUKSO's Node Requirements
+
+Here is what you will need to run a Lukso node :
+
+- A stable Internet connection. The longer you stay online, the better your rewards. A spotty Internet connection will hurt your returns, and by extension, the LYX ratio growth.
+- At least 10Mbps of bandwidth both up and down. A LUKSO node usually takes around 8Mbps to 10Mbps up & down of network traffic, depending on your configuration and number of nodes you decide to run.
+- No data cap imposed by your ISP. It is not suggest to run a node if your Internet plan comes with a monthly data cap.
+- Stable electricity. For the same reason as needing a stable Internet connection, you also want to have reliable power. This can be mitigated with a large UPS (backup battery) to deal with short blackouts.
+- A computer or Virtual Machine with sufficient specs. The computer can be a local machine, or it can be a Virtual Private Server (VPS) hosted in the cloud.
+
+The following specifications are considered minimum requirements:
+
+Linux or macOS Operating System
+
+- Quad core CPU (or dual-core hyperthreaded); both `x64` and `arm64` are supported
+- 8 GB of RAM (preferably DDR4); Typical configurations tend to use 16 or 32 GB of RAM
+- 2 TB of free SSD Disk Space
+- A spinning platter hard drive is generally not fast enough to handle the constant random reads and writes that blockchain activity requires; you MUST use a solid state drive.
+
+## Installing the LUKSO CLI
+
+The LUKSO CLI serves the following purposes:
+
+- easy installation of all node types (All installations are done directly into /bin/, and you will not need any docker container)
+- easy start and stopping of local nodes (Since LUKSO will run as a daemon)
+- easy access to nodes logs
+
+We abstracted all the complicated steps to onboard new validators as easy as possible. The process is fast and with only a few command lines you can get started.
+
+Here are the steps:
+
+### Install Dependencies
+
+1. [curl](https://curl.se/)
+
+#### Install curl
+
+```sh
+sudo apt-get -y update
+sudo apt-get -y install curl
+```
+
+#### Install LUKSO
+
+On a Linux or MacOS open the terminal and copy the following line of code to install the LUKSO client
+
+```console
```

Comment:
console or sh?

Let's keep either one for the all docs, I think sh is good

Reply:
changed all to sh